packages/components/src/theme/ComposerTheme/Components/Slider.ts

```diff
@@ -5,6 +5,14 @@ const sharedSizeStyle = (props) => ({
   thumb: {
     w: '1rem',
     h: '1rem',
+    _active: {
+      w: '1.125rem',
+      h: '1.125rem',
+    },
+    _focus: {
+      w: '1rem',
+      h: '1rem',
+    },
```

Comment:
this change breaks the Slider with a bigger thumb, visit https://5eed577398d08d0022b52d76-cwbpjoawuk.chromatic.com/?path=/story/components-primitives-slider--with-custom-thumb and change the value of the slider for details

Reply:
According to design we should preseve the scalation of chakra for active/focus interactions so I've removed this custom active/focus styles